eventstore/query_test.go

```diff
@@ -142,6 +143,16 @@ func TestInvalidSortField(t *testing.T) {
 	}
 }
 
+func TestInvalidSliceType(t *testing.T) {
```

Comment:
Extra test to check the new feature of checking that received slice for `Find` controls that has the right type for the queried Model.